src/math/src/sha512.cairo

```diff
@@ -8,6 +8,23 @@ pub const SHA512_LEN: usize = 64;
 
 pub const U64_BIT_NUM: u64 = 64;
 
+// Powers of two to avoid recomputing
+pub const TWO_56: u64 = 0x100000000000000;
+pub const TWO_48: u64 = 0x1000000000000;
+pub const TWO_40: u64 = 0x10000000000;
+pub const TWO_32: u64 = 0x100000000;
+pub const TWO_24: u64 = 0x1000000;
+pub const TWO_16: u64 = 0x10000;
+pub const TWO_8: u64 = 0x100;
+pub const TWO_4: u64 = 0x10;
+pub const TWO_2: u64 = 0x4;
+pub const TWO_1: u64 = 0x2;
+pub const TWO_0: u64 = 0x1;
+
+// Max u8 and u64 for bitwise operations
+pub const MAX_U8: u64 = 0xff;
+pub const MAX_U64: u128 = 0xffffffffffffffff;
```

Comment:
Can't you use `core::interger::BoundedU8::max` rather than defining those again?
they are always inlined so it shouldn't have a cost.

Reply:
Oh it looks like it was using Bounded::max before and you change it to this. Can you explain the perf improvement from it?

Reply:
No perf improvement, just *aesthetics*.
Using that was breaking up some lines of code.
With smaller yet descriptive vars it stayed pretty.